src/wallet/rpcwallet.cpp

```diff
@@ -2432,6 +2434,41 @@ UniValue getwalletinfo(const JSONRPCRequest& request)
     return obj;
 }
 
+UniValue listwallets(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0)
+        throw std::runtime_error(
+            "listwallets\n"
+            "Returns a list of currently loaded wallets.\n"
+            "For full information on the wallet, use \"getwalletinfo\"\n"
+            "\nResult:\n"
+            "[\n"
+            "  \"walletname\": xxxxx,             (string) the wallet name\n"
```

Comment:
Should drop the `:xxxx,` here since an array is just a list of items, not key/value pairs.